LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2006-2018 Doctrine Project
+Copyright (c) 2006-2019 Doctrine Project
```

Comment:
Probably better to just drop the year overall

Reply:
Some people think that 2006 was the beginning of the project.
It's good to always be mentioned this year as the end.

Reply:
Do you mean to remove the end of this year in general?

Reply:
I'd remove the whole range - it is something that can be always extracted via our signed releases and commits.

Reply:
Unresolving because the range is still here… did you forget to push your changes, maybe?

Reply:
I did not get you.
@greg0ire

Reply:
That conversation was marked as resolved, but the issue is still here. You may mark it as resolved once this is fixed, like so:
```suggestion
Copyright (c) Doctrine Project
```

Reply:
Oh, yes, my changes have not been saved.
I added the changes again.
@greg0ire  Thank you.